img/u/uwaterloo.ca.svg

```diff
@@ -0,0 +1 @@
 0 .1 0 .1-.1.1-.1.2-.2.2-.3.2 0 .4 0 .5-.1-.1-.2 0-.3 0-.4m-3.5-15.2c-1.3.4-2.1 1.6-2.5 2.7 0 .1-.1.2-.1.5 0 .1.8-.8 1.3-.5 0 .1-.1.7-.3.8.7-.1 1.6-.2 2.1-.7.1-.1 0 .5.1.7.3.7.9 1.2 1.5 1.6.2.1.1-.3.5-.6.3-.2.7.2 1 .3.2.1-.2-.6-.3-.8-.4-2 .4-3.6.4-5.5 0-.3-.1-.9-.5-1.2-.4 1.6-1.9 2.3-3.2 2.7m-11-31.4c-.4.2-1.4.3-1.2-.1.2-.5.8-.6 1.2.1m.9-.7c-.6 0-1.1-.5-1.6-.3-.6.2-1.3.5-1.9.7-.2.1-.7.3-.6.8.6-.2 1.5-.6 1.9.1.2.4.2.8 0 1.4-.1.4-.9.6-1.5.5.1.1.2.3.1.5 1.3-.3 3.2.2 4.1-1.3.3-.5-.1-1 0-1.5.1-.2.4-.4.5-.7.1-.2-.2-.5-.2-.7-.2.3-.4.5-.8.5m24.2.5c-.5.2-1.7.6-1.5.1.3-.7 1-.4 1.5-.1 0 0 .1-.1 0 0m.8 2.3c.5-.3.6-1 .8-1.5.1-.3.5-.3.7-.7v-.6c-.3.2-.9.4-1.3.3-1.2-.3-1.6-1-2.8-.1-.3.3-1 0-1.5.6-.1.1-.1.3-.1.5.6 0 1.5-.7 1.9.1.1.1.1.2.1.4v.6c0 .2-.1.4-.3.6.1-.2.1-.2 0-.1l-.1.1c-.1.1-.2.1-.2.1-.1 0-.1 0-.2.1H41.2c-.2 0-.2-.1-.3 0-.1.1-.1.2-.1.3.1 0 .2 0 .3.1.1.1.3.1.4.1h1.3c.2 0 .1 0 .3-.1.1 0 .2-.1.3-.1.1-.1.2-.2.4-.3.2-.1.5-.1.7-.3M29.7 34.4c-.1 0-.1 0-.2-.1s-.1-.2-.1-.3c0-.1.1-.1.1-.2s.1 0 .2-.1l.1-.1h.1c.1 0 .2.1.4.1s.3 0 .5.1l-.1.1c-.2.3-.8.6-1 .5m1.6-.9c-.8-.2-1.5-.4-2.1 0 0 0-.5.5-.4.5-.9 0-1.5.6-1.4 1 .3-.2.7-.5 1-.2.3.2.5 1 .3 1.4-.1.2-.4.6-.9.6-.2 0-.1.3-.1.4 1.2-.1 2.6-.8 3.6-1.4.5-.3.5-.8.7-1.4.1-.4.8-.5.6-1.2-.2.4-.8.4-1.3.3M48 28.3c0-.2.4-.7.5-1.1 0-1.4.8-2 .9-3.4 0-.4-1.5-.7-1.6-1.5 0-.4-.2-1 .2-1 .5 0 1-.1 1.4-.4 1.6-1.4.2-3.4.1-5.1 0-.4.7-1.2.6-1.5-.1-1.5-.4-2.8.1-4.2-.4.4-.5 1-.9 1.3-.2.1-.2-.2-.2-.2.2-1.3.7-2.4 1.3-3.6.7-1.4.3-2.6-.9-3-1.3-.6-2.4.5-2.5 1.6-.1 1 .2 2.3.5 3.1.1.2 0 .5.1.7.2-.9.2-1.8.2-2.7 0-.2.2-.5.4-.6.8-.2 1.2.3 1.1.8-.2 1.4-.4 2.4-.8 3.7-.4 1.5.2 3.1.3 4.6-.6-.4-1.2-.4-1.5.5-.1.5.3 1 .5 1.4 0-.6.6-1.4 1.1-1 .7.4.7 2.5.5 2.8-.3.6-.8 1-1.6.8-.4-.1-.2-.7-.5-1-1.3-1.4-1.9-3-2-4.9 0-.3.1-1 .1-1-1.7-.7-2.5 0-3.2 1-.1-.3-.8-.7-.8-.6-.3.6-.2 1.7.3 2.1 1.3 1.1 2.6 1.9 3.4 3.5-.8.1-1.6.3-1.9 1.2-.3.9-.2 2.1.7 2.4.8.3 2 0 2.3 0 .8.1 1.7.8 1.5 1.5 0 0-.3 1.8-1.1 1.4-.4-.2-.2-1.2-.8-1.4-.8-.2-1 1-.9.9.3-.3.5.2 1 .5-1.8-.4-1.2 1.2-1.2 1.1.1-.2.4 0 1 0 0 0-.1.1-.2.1-.1.2-.2.2-.2.4v.2c0 .1 0 .2.1.2 0 0 0-.1.1-.1h.1s.1 0 .2.1h.4c.1 0 .2 0 .3-.1l.1-.1c.1-.2.2-.2.2-.2.2-.4.3-.4.8-.3l.1.1s.1.3.1.2c.1.5.2.6.2.8m-12.5-16c.1.1-.1.2-.2.1-.8-.9-2.1-.8-2 .7.5-.2.6.4.9.2.3-.2.5-.4.8-.2.2.1.1.4 0 .4-1.4.5-.3 1.4-.5 1.6.2-.2.8-.2 1.3-1 .1-.2.8-.2 1.1-.1.4.1 0 .7.2.7.4-.1 1.3.4 1.7.8.8.8 1.5.7 2.5 1.2-.7-.6-.7-1.7-.5-2.5-.9 0-1.7-.2-2.5-.4-1.6-.4-1.5-2.1-1.8-2.7l-.1-.1c-.1-.3-.4-.4-.7-.4H35.4c.1 0 .1.1 0 .2v.2c-.2.2-.3.5-.2.7 0 .1 0 .1.1.1 0 0 0 .1.1.1.1.1.1.2.2.3l-.1.1z"/><path d="M52.9 5.7c-.2 9.6-.4 18.8-3.2 27.7L29.8 17.6 9.9 33.4c-2.8-8.9-3-18.1-3.2-27.7 0-.9 0-1.8-.1-2.7h46.3v2.7m-8.7 39.1C41 49.8 36 54 29.8 57c-6.2-2.9-11.2-7.1-14.5-12.1-.6-.8-1.1-1.7-1.5-2.5l16-12.7 16 12.7c-.5.7-1 1.6-1.6 2.4m-33.3-8.7 19-15 18.9 15c-.5 1.2-1 2.5-1.6 3.7L29.8 26.1 12.5 39.8c-.6-1.2-1.2-2.4-1.6-3.7M5.7 2v.5c0 1.1.1 2.1.1 3.2.3 13.9.6 27 8.8 39.7 3.4 5.2 8.6 9.5 15.1 12.5l.2.1.2-.1c6.5-3 11.7-7.3 15-12.5 8.2-12.6 8.5-25.8 8.8-39.7 0-1.1 0-2.1.1-3.2V2H5.7z"/></svg>
```

Comment:
Another one which could do with a raster-image replacement.
![png](https://user-images.githubusercontent.com/3535780/179313561-646c794b-3ec2-43ff-bc39-77b58a8596bd.png)

Reply:
Changed from svg to png in https://github.com/2factorauth/twofactorauth/pull/6860/commits/b532efef7464940bcba80c5cd77c98bd7885da4e